### PR TITLE
Log messages for aws-ec2-pubkeys

### PR DESCRIPTION
**Pull request by wraithm**
Log what's failing and what we're scanning

### CODE REVIEW

File: src/AWS/Types.hs [hunk 1]
```diff
@@ -15,6 +15,12 @@ import Data.Aeson.Types (typeMismatch)
 import Data.Text (Text)
 
 
+type Fqdn = Text
```
Comment (bradsherman) [inline src/AWS/Types.hs:18]: what is `Fqdn`?

Comment (wraithm) [reply src/AWS/Types.hs:18]: Fully Qualified Domain Name

Comment (wraithm) [reply src/AWS/Types.hs:18]: https://en.wikipedia.org/wiki/Fully_qualified_domain_name

File: src/AWS/PubKeys.hs [hunk 2]
```diff
@@ -35,16 +36,28 @@ import Text.Printf (printf)
 import Prelude hiding (filter, takeWhile)
 
 
-getPubKeys :: [Ec2Instance] -> IO [Ec2Instance]
-getPubKeys = fmap catMaybes <$> mapConcurrently getPubKey
+getPubKeys :: [Ec2Instance] -> IO [Either Ec2Instance Ec2Instance]
```
Comment (bradsherman) [inline src/AWS/PubKeys.hs:39]: why `Either Ec2Instance Ec2Instance`?

Comment (wraithm) [reply src/AWS/PubKeys.hs:39]: Left is failure, right is success.

You get all of the data associated with the one that failed in the case it failed.

**Pull request merged**